### PR TITLE
added enum Bank

### PR DESCRIPTION
**Pull request by marvel-hero2**

### CODE REVIEW

File: src/main/java/com/app/telegram/bank/Bank.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.app.telegram.bank;
 
 public enum Bank {
-Pryvat
+Pryvatbank,
+Monobank
```
Comment (russtrUA) [inline src/main/java/com/app/telegram/bank/Bank.java:5]: Please add NBU

**Review by russtrUA — COMMENTED** [review]
Please add code for banks as a field and create a constructor

**Comment by marvel-hero2** [comment]
added code for banks

**Review by marvel-hero2 — COMMENTED** [review]
added code for banks

**Review by russtrUA — APPROVED** [review]
Done

**Pull request merged**